src/validator-registry-tp/src/validator_registry_tp.rs

```diff
@@ -204,10 +204,10 @@ impl ValidatorRegistryTransactionHandler {
                     originator_public_key_hash: &String,
                     val_reg_payload: &ValidatorRegistryPayload,
                     context: &mut TransactionContext,) -> Result <(), ValueError> {
-         let signup_info : ValidatorRegistrySignupInfo = serde_json::from_str(val_reg_payload.signup_info_str).unwrap();
+         let signup_info : ValidatorRegistrySignupInfo = serde_json::from_str(&val_reg_payload.signup_info_str).unwrap();
          let proof_data : SignupInfoProofData = serde_json::from_str(&*signup_info.proof_data).unwrap();
 
-         let report_public_key_pem = self._get_config_setting(context, "sawtooth.poet.report_public_key_pem").unwrap();
+         let report_public_key_pem = self._get_config_setting(context, &"sawtooth.poet.report_public_key_pem".to_string()).unwrap();
```

Comment:
I see in PoET2 core, these settings have "sawtooth.consensus.poet2." prefix but we are using "sawtooth.poet." prefix (same as what followed in PoET 1). It's better to keep them consistent if possible.

Reply:
I see same code in original poet2 repo

Reply:
It seems the repo switch happened before those changes could be pushed.

Reply:
Ok, no issues. Please push those changes to new repo. So copy happened correct but code was not pushed.